contrib/cmake/build3rdparty.bash

```diff
@@ -1,17 +1,80 @@
-#!/bin/bash
-# The usual way to use this file is to make a new folder 3rdParty next to your sdk folder, copy this script to it, and run it there.
-# Then building via cmake can refer to these libraries.
-
-git clone https://github.com/Microsoft/vcpkg.git
-cd vcpkg
-./bootstrap-vcpkg.sh
-./vcpkg install --triplet x64-linux zlib
-./vcpkg install --triplet x64-linux cryptopp
-./vcpkg install --triplet x64-linux openssl
-./vcpkg install --triplet x64-linux c-ares
-./vcpkg install --triplet x64-linux curl    
-./vcpkg install --triplet x64-linux sqlite3
-./vcpkg install --triplet x64-linux gtest
-#./vcpkg install --triplet x64-linux libsodium
+# This script is for getting and building the 3rd party libraries that the MEGA SDK uses (some are optional, and some are only needed only by MEGA apps too)
+# 
+# Your 3rdParty library builds should be outside the SDK repo.  We are moving to use vcpkg to build most of them. You can start it like this:
+#
+# mkdir 3rdParty
+# cd 3rdParty
+# git clone https://github.com/Microsoft/vcpkg.git
+# cd vcpkg
+# .\bootstrap-vcpkg.sh -disableMetrics
+#
+# Comment out any libraries that you won't use.
+# If using pdfium, follow the instructions below to get the source code
+#
+# On Mac, go to ports/libraw/CONTROL and comment out the freeglut dependency
+# 
+# From your 3rdParty/vcpkg folder, run this script (in its proper location) with the desired triplet as the parameter.  (usually x64-linux or x64-osx)
 
+export TRIPLET=$1
+
```

Comment:
echo "" > buildlog

Reply:
will add

Reply:
is this to clean prior build records from the buildlog?  Those can be handy to see, eg if you have to run the script several times after adjusting for things that are not working, eg. apt-get install various things (already completed vckpg packages will be skipped by vcpkg).   How about we add a prominent line saying "started build" and the date/time? thanks

Reply:
Yes, the purpose is to clean prior build records.. I am ok with an info line, up to you. No problem at all.